adapters/tappx/params_test.go

```diff
@@ -60,4 +65,13 @@ var invalidParams = []string{
 	`{"tappxkey": 1, "endpoint":"ZZ1INTERNALTEST149147915", "host":""}`,
 	`{"tappxkey":"pub-12345-android-9876", "endpoint": 1, "host":""}`,
 	`{"tappxkey": 1, "endpoint": 1, "host": 123}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
+	`{"tappxkey": "1", "endpoint": 1}`,
+	`{"tappxkey": "1", "endpoint": "1", "mktag":1}`,
+	`{"tappxkey": "1", "endpoint": "1", "mktag":[1,2]}`,
```

Comment:
Can you add `"host":"host"` to each of these two test cases where `mktag` is an invalid type to isolate that it is `mktag` that is causing the error and not a missing host?

Reply:
Yeep, thanks!
Change in 8d09494.